catalog/ui/src/app/Catalog/CatalogItemFormReducer.ts

```diff
@@ -339,12 +348,22 @@ function reduceFormStateTermsOfServiceAgreed(initialState: FormState, termsOfSer
   };
 }
 
-function reduceFormStateWorkshop(initialState: FormState, workshop: WorkshopProps = null): FormState {
+function reduceFormStateWorkshop(
+  initialState: FormState,
+  allowServiceNamespaces: ServiceNamespace[],
+  serviceNamespace: ServiceNamespace,
+  workshop: WorkshopProps = null
+): FormState {
   const isSalesforceIdRequired = salesforceIdRequired({ ...initialState, workshop });
+  const resetServiceNamepace = true;
+  allowServiceNamespaces && !(
```

Comment:
i guess that this is not correct, you might wanted to assign this variable to the boolean above?

Reply:
Good catch. I had put in a change here to force the value to test and didn't fix it.